bitnami/grafana-alloy/values.yaml

```diff
@@ -0,0 +1,931 @@
+    ##
+    updatePolicy:
+      updateMode: Auto
+  ## @param autoscaling.hpa.enabled Enable HPA for Grafana Alloy pods
+  ## @param autoscaling.hpa.minReplicas Minimum number of replicas
+  ## @param autoscaling.hpa.maxReplicas Maximum number of replicas
+  ## @param autoscaling.hpa.targetCPU Target CPU utilization percentage
+  ## @param autoscaling.hpa.targetMemory Target Memory utilization percentage
+  ##
+  hpa:
+    enabled: false
+    minReplicas: ""
+    maxReplicas: ""
+    targetCPU: ""
+    targetMemory: ""
+
+## @section Grafana Alloy parameters
+
+alloy:
+## Bitnami Grafana Alloy image version
+## ref: https://hub.docker.com/r/bitnami/grafana-alloy/tags/
+## @param alloy.image.registry [default: REGISTRY_NAME] Grafana Alloy image registry
+## @param alloy.image.repository [default: REPOSITORY_NAME/grafana-alloy] Grafana Alloy image repository
+## @skip alloy.image.tag Grafana Alloy image tag (immutable tags are recommended)
+## @param alloy.image.pullPolicy Grafana Alloy image pull policy
+## @param alloy.image.pullSecrets Grafana Alloy image pull secrets
+## @param alloy.image.debug Enable image debug mode
+##
+  image:
+    registry: docker.io
+    repository: bitnami/grafana-alloy
+    tag: 1.8.3-debian-12-r0
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ## ref: https://kubernetes.io/docs/concepts/containers/images/#pre-pulled-images
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ## e.g:
+    ## pullSecrets:
+    ##   - myRegistryKeySecretName
+    ##
+    pullSecrets: []
+    debug: false
+  ## @param alloy.containerPorts.http Grafana Alloy HTTP container port
+  ##
+  containerPorts:
+    http: 8080
+  ## @param alloy.extraContainerPorts Optionally specify extra list of additional ports for Grafana Alloy containers
+  ## e.g:
+  ## extraContainerPorts:
+  ##   - name: myservice
+  ##     containerPort: 9090
+  ##
+  extraContainerPorts: []
+  ## @param alloy.existingConfigmap The name of an existing ConfigMap with your custom configuration for Grafana Alloy
+  ##
+  existingConfigmap: ""
+  ## @param alloy.configuration Specify content for Grafan Alloy config file. Omitted if alloy.existingConfigmap is provided.
+  ## e.g:
+  ## configuration: |-
+  ##   ...
+  ##
+  configuration: ""
+  ## @param alloy.extraConfig Append extra configuration to the default config file
+  ##
+  extraConfig: ""
+  ## @param alloy.resourceType Type of controller to use for deploying Grafana Alloy in the cluster.
+  ## Must be one of 'daemonset', 'deployment', or 'statefulset'.
+  ##
+  resourceType: 'daemonset'
+
+  ## @param alloy.replicaCount Number of pods to deploy. Ignored when alloy.resourceType is 'daemonset'.
+  ##
+  replicaCount: 1
```

Comment:
These two parameters also affect ConfigMap Reload. We should rename them to `resourceType` and `replicaCount` given they don't affect Grafana Alloy containers exclusively.